packages/sampler/stories/default/ViewManager.js

```diff
@@ -37,6 +42,27 @@ const ViewManagerLayout = (props) => {
 		setSelected(ev.selected);
 	}, [setSelected]);
 
+	const selectedEnd = select('end', prop.end, ViewManagerConfig, prop.end[0]);
+	const selectedStart = select('start', prop.start, ViewManagerConfig, prop.start[0]);
```

Comment:
```suggestion
	const selectedStart = select('start', Object.keys(prop.start), ViewManagerConfig, prop.start[0]);
```

Can't we use object for end and start?
end : {'index':0, 'index+1':1, 'index+2':2},

Reply:
Thanks. I've changed to use object for them.